contracts/interfaces/IAxelarForecallable.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.9;
```

Comment:
Same

Reply:
Yeah I noticed that and decided to fix the pragma in a separate PR #6 because of amount of files affected

Reply:
Reverting those lines for clarity